docs/api.md

```diff
@@ -0,0 +1,1683 @@
+| `revoked` | boolean | If this key has been disabled for use. |
+
+## `BlockObject`
+
+The JSON representation of a key on the Flow blockchain.
+| Key | Value Type | Description |
+| ---- | ---------- | ----------- |
+| `id` | string | The id of the block. |
+| `parentId` | string | The id of the parent block. |
+| `height` | number | The height of the block. |
+| `timestamp` | object | Contains time related fields. |
+| `collectionGuarantees` | [[CollectionGuaranteeObject](#CollectionGuaranteeObject)] | Contains the ids of collections included in the block. |
+| `blockSeals` | [SealedBlockObject] | The details of which nodes executed and sealed the blocks. |
+| `signatures` | Uint8Array([numbers]) | All signatures. |
+
+## `BlockHeaderObject`
+
+The subset of the [BlockObject](#BlockObject) containing only the header values of a block.
+
+| Key         | Value Type | Description                   |
+| ----------- | ---------- | ----------------------------- |
+| `id`        | string     | The id of the block.          |
+| `parentId`  | string     | The id of the parent block.   |
+| `height`    | number     | The height of the block.      |
+| `timestamp` | object     | Contains time related fields. |
+
+## `CollectionGuaranteeObject`
+
+A collection that has been included in a block.
+
+| Key            | Value Type        | Description          |
+| -------------- | ----------------- | -------------------- |
+| `collectionId` | string            | The id of the block. |
+| `signatures`   | [SignatureObject] | All signatures.      |
+
+## `CollectionObject`
+
+A collection is a list of transactions that are contained in the same block.
+| Key | Value Type | Description |
+| ---- | ---------- | ----------- |
+| `id` | string | The id of the collection. |
+| `transactionIds` | [string] | The ids of the transactions included in the collection. |
+
+## `ResponseObject`
+
+The format of all responses in FCL returned from `fcl.send(...)`. For full details on the values and descriptions of the keys, view [here](https://github.com/onflow/flow-js-sdk/tree/master/packages/sdk/src/response).
+| Key |
+| ---- |
+| `tag` |
+| `transaction` |
+| `transactionStatus` |
+| `transactionId` |
+| `encodedData` |
+| `events` |
+| `account` |
+| `block` |
+| `blockHeader` |
+| `latestBlock` |
+| `collection` |
+
+## `Event Object`
+
+| Key | Value Type | Description |
+| ---- | ---------- | ----------- |
+| `blockId` | string | ID of the block that contains the event. |
+| `blockHeight` | number| Height of the block that contains the event. |
+| `blockTimestamp` | string | The timestamp of when the block was sealed in a `DateString` format. eg. `'2021-06-25T13:42:04.227Z'` |
+| `type` | [EventName](#EventName) | A string containing the event name. |
+| `transactionId` | object | :tomato: TODO |
+| `transactionIndex` | object | :tomato: TODO |
+| `eventIndex` | object | :tomato: TODO |
```

Comment:
```suggestion
| `eventIndex` | object | Used to prevent replay attacks. |
```

Reply:
For these 'eventIndex (number), transactionIndex (number), and transactionId (string)' fields, I think the type also needs to be changed from 'object' to 'string' or 'number' - my mistake but would you mind changing those as well

Reply:
OK. Updated.